### PR TITLE
Remade Rmd to work with Synapse tables

### PR DESCRIPTION
**Pull request by jaybee84**
Currently has public data only.

### CODE REVIEW

File: figs/Scidata2019_figs_genomicVariants.Rmd [hunk 1]
```diff
@@ -129,18 +114,16 @@ color_list <- c(allcolors, morecolors1, morecolors2)
 ## Oncoplot showing all the samples and all genes that are important for cNF and MPNST
 
 require(maftools)
-# Make "maf object" to work with maftools for easy visualization 
-jhu_maf <- merge_mafs(c(jhu_anno_new$filepath), verbose = TRUE, clinicalData = jhu_anno_new) 
 
 # Make oncoplot for all samples for specific genes (top column bar shows mutational burden of the samples, row bar on the right shows frequency of mutations per gene)
 png("Scidata_fig_1_allsamples_oncoplot.png",height = 800, width = 1800 )
```
Comment (sgosline) [inline figs/Scidata2019_figs_genomicVariants.Rmd]:
```suggestion
pdf("Scidata_fig_1_allsamples_oncoplot.pdf")#,height = 800, width = 1800 )
```

File: figs/Scidata2019_figs_genomicVariants.Rmd [hunk 2]
```diff
@@ -154,22 +137,21 @@ We were able to crosscheck other samples in the bank with their clinical sequenc
...
-
+sample_2031 <- jhu_public_clindata[(jhu_public_clindata$individualID == "2-031" & jhu_public_clindata$specimenID != "2-031 Xenograft"),]
 
 #Make Maf file for sample 2-031
-jhu_sample_2031 <- merge_mafs(c(sample_2031$filepath), verbose = TRUE, clinicalData = sample_2031) 
+jhu_sample_2031 <- subsetMaf(jhu_public_maf_all, tsb = c(sample_2031$Tumor_Sample_Barcode)) 
 
 # Make oncoplot to show the SNPs and mutation of the samples in various genes of interest
 print("Fig 1 : Sample 2-031")
 png("Sample_2-031.png", width=800, height = 800)
```
Comment (sgosline) [inline figs/Scidata2019_figs_genomicVariants.Rmd]:
```suggestion
pdf("Sample_2-031.pdf")#, width=800, height = 800)
```

File: figs/Scidata2019_figs_genomicVariants.Rmd [hunk 3]
```diff
@@ -154,22 +137,21 @@ We were able to crosscheck other samples in the bank with their clinical sequenc
...
 oncoplot(jhu_sample_2031, genes = c("NF1","PIK3CA", "PRC2","MEK1", "MEK2", "SUZ12", "ASTE1", "EGFR", "LRP1B", "MSH3", "PDCD1", "S1PR2", "EED", "EZH2"),
```
Comment (sgosline) [inline figs/Scidata2019_figs_genomicVariants.Rmd]: I'm not sure why the other sample names show up in this figure when they're not there. any ideas? we can remove manually if need be.

Comment (jaybee84) [reply figs/Scidata2019_figs_genomicVariants.Rmd]: Not quite sure... looking into it..

**Pull request merged**